core/app/models/mno_enterprise/base_resource.rb

```diff
@@ -61,16 +61,16 @@ def update_attribute(name, value)
     # emulate active record call of callbacks
     def save(*args)
       run_callbacks :save do
```

Comment:
We should differentiate between create and update on the `save` method.

See: https://github.com/rails/rails/blob/310918f6a194bf5752fe1025930881756f5d8a8e/activerecord/lib/active_record/persistence.rb#L128

Based on this we should call the `:save` and `:create` or `:update` callbacks